sdk/java-sdk/src/main/scala/kalix/javasdk/impl/workflow/WorkflowImpl.scala

```diff
@@ -38,12 +38,12 @@ import kalix.javasdk.impl.workflow.WorkflowEffectImpl.TransitionalEffectImpl
 import kalix.javasdk.impl.workflow.WorkflowEffectImpl.UpdateState
```

Comment:
WorkflowEntityImpl?

Reply:
if yes, then I suppose we need to change also other classes in `/impl/workflow` package, and actually rename the package as well?

Reply:
yep, I will do it.